documentation/modules/oauth/con-oauth-authorization-keycloak-example.adoc

```diff
@@ -100,21 +136,21 @@ Roles cannot be stored in an LDAP identity provider.
 If LDAP is a requirement, you can use groups instead, and add Keycloak roles to the groups so that when users are assigned a group, they also get a corresponding role.
```

Comment:
```suggestion
If LDAP is a requirement, you can use groups instead, and add Keycloak roles to the groups so that when users are assigned a group they also get a corresponding role.
```